view/template/trial.hbs

```diff
@@ -69,6 +69,16 @@
         </div>
         <div class="col-sm-12 col-md-7 col-lg-6 col-xl-6">
             <div class="card">
+                    <input type="hidden" id="trialId" name="trialId" value="{{trial.id}}">
```

Comment:
I need a hidden field to get the `trial.id`. If I put it in the form then it is causing issues. There arises a need to define that field in `router` file as well.